tests/lib/nsxt_mocktest.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2023 The Capirca Project Authors All Rights Reserved.
+# Copyright 2023 VMware, Inc. SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Nsxt Mock Test terms for nsxt module."""
+
+INET_TERM = """\
+  term permit-mail-services {
+    destination-address:: MAIL_SERVERS
+    protocol:: tcp
+    destination-port:: MAIL_SERVICES
+    action:: accept
+  }
+  """
+
+INET6_TERM = """\
+  term test-icmpv6 {
+    protocol:: icmpv6
+    icmp-type:: echo-request echo-reply
+    action:: accept
+  }
+  """
+
```

Comment:
Some (or all) of the tests should specify a security group in inputs and outputs, so that we can be sure security group gets used.